lms/djangoapps/verify_student/models.py

```diff
@@ -1140,6 +1145,9 @@ class VerificationStatus(models.Model):
     response = models.TextField(null=True, blank=True)
     error = models.TextField(null=True, blank=True)
 
+    # cache key format e.g verification.<username>.<course_key> = '{verification.1.edx/demo/course}'
+    VERIFICATION_STATUS_CACHE_KEY = u"verification.{}.{}"
```

Comment:
minor: since this is only referenced by the classmethod `cache_key_name`and not exposed directly anywhere else, a "public" class variable doesn't seem necessary.